Find a functionally equivalent code:android.graphics.Canvas.drawRect:COMMENT
Method Modifier: public      
Comment:/**
 * Draw the specified Rect using the specified paint. The rectangle will
 * be filled or framed based on the Style in the paint.
 *
 * @param left   The left side of the rectangle to be drawn
 * @param top    The top side of the rectangle to be drawn
 * @param right  The right side of the rectangle to be drawn
 * @param bottom The bottom side of the rectangle to be drawn
 * @param paint  The paint used to draw the rect
 */

@@ -1,3 +1,3 @@
 {
-    native_drawRect(mNativeCanvas, left, top, right, bottom, paint.mNativePaint);
+    native_drawRect(mNativeCanvasWrapper, left, top, right, bottom, paint.mNativePaint);
 }

